Python lib docs (#1)
lzpap/documentation: client-libraries/0.1/how-to-guides/python/read-transactions.md

# Read transactions on the Tangle in Python

**In this guide, you get [transactions](root://getting-started/0.1/transactions/transactions.md) from the Tangle by connecting to a [node](root://getting-started/0.1/network/nodes.md) and asking it to filter them by their bundle hash. Then, you decode the message in the transaction and print it to the console.**

## Packages

> To complete this guide, you need to install the following package (if you're using Go modules, you just need to reference this package):

Reviewer: ```suggestion
To complete this guide, you need to install the following package:
```

```bash
pip install pyota
```

## IOTA network

In this guide, we connect to a node on the [Devnet](root://getting-started/0.1/network/iota-networks.md#devnet).

## Code walkthrough

1. Import the packages

    ```python
    from iota import Iota
    from iota import Transaction
    ```

2. Connect to a node

    ```python
    api = Iota('https://nodes.devnet.iota.org:443', testnet = True)
    ```

3. Define the bundle hash that you want to use to filter transactions 

    ```python
    bundle = 'IYPHGPIAO99XFAIBRXB9BEQLTZBCXTAGHUXL9UUXGGHHNKEBVEANQIBOALKSO9KLHTEEZXXPB9IOBK9RB'
    ```

> 4. Use the [`find_transaction_objects()`](https://github.com/iotaledger/iota.py/blob/master/docs/api.rst#find_transaction_objects) method to get transactions by the value of their `bundle` field

Reviewer: Link to https://pyota.readthedocs.io/en/latest/api.html#find-transaction-objects
```suggestion
4. Use the [`find_transaction_objects()`](https://pyota.readthedocs.io/en/latest/api.html#find-transaction-objects) method to get transactions by the value of their `bundle` field
```

>     ```go

Reviewer: ```suggestion
    ```python
```

>     transactions = api.find_transactions(bundles=[bundle])

Reviewer: ```suggestion
    transactions = api.find_transactions_objects(bundles=[bundle])
```

    ```

5. Convert the message in the bundle's first transaction to ASCII characters and print it to the console

    ```python

>     message = transactions['transactions'][0].signature_message_fragment

Reviewer: ```suggestion
    # message is encoded into trytes
    message_trytes = transactions['transactions'][0].signature_message_fragment
```
